fortran/testpar/Makefile.am

```diff
@@ -36,7 +36,7 @@ TEST_PROG_PARA=parallel_test subfiling_test async_test
 check_PROGRAMS=$(TEST_PROG_PARA)
 
 # Temporary files
-CHECK_CLEANFILES+=parf[12].h5 subf.h5*
+CHECK_CLEANFILES+=parf[12].h5 h5*_tests.h5 subf.h5* test_async_apis.mod
```

Comment:
@brtnfld  - it looks like this is created during `make check` and not `configure` so I added it to the clean files